### ISSUE TITLE
IDEA: Clone Fact / Update Document?

### ISSUE THREAD

**Issue by ghost** [idmc-labs/idmc-database-feedback#335]
It would be great to be able to clone a fact then change the source document. For example this would be useful in cases where we have added many geographic locations to one fact, and would have to repeat this process for another fact from a difference source document.

**Comment by allochi** [comment]
Are you talking about templating? wouldn't it be more useful if I allow to save a generic pre-populated fact template that you can choose and fill in missing information? like if you find yourself having 4-5 repetitive patterns, you can save a generic fact template and recall one, fill it and save it.

**Comment by ghost** [comment]
Yeah, you can have a pre-populated fact... the main thing is to be able to update the source document.  So thinking that you can clone a fact that has most of the information you need, and then just point it to a different source doc. 

For example, for the Philippines, I have an updated figure for a disaster event... rather than create a new doc, then create a new fact... to find the previous fact with the same information (ex. all the locations), clone it, and then attach the updated source doc.
